CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ TRIBITS_PACKAGE_DECL(DataTransferKit)
 TRIBITS_ADD_DEBUG_OPTION()
 TRIBITS_ADD_SHOW_DEPRECATED_WARNINGS_OPTION()
 
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror")
+
```

Comment:
No.  Configure with `-D CMAKE_CXX_FLAGS=-Werror` if you like but do not add it here.  We certainly do not want to pass the option for all DTK builds.
That said, I am fine with adding it to `scripts/docker_cmake`.